Assets/Scripts/LevelPhysics/Floor.cs

```diff
@@ -0,0 +1,34 @@
+using UnityEngine;
+
+public class Floor : MonoBehaviour
+{
+   [SerializeField] private float _speed = 1.5f;
+    private bool _dir = true;
+
+    // Update is called once per frame
+    void Update()
+    {
+        Move(_dir);
+    }
+
+    private void Move( bool directionMovement)
+    {
+        if (directionMovement)
+        {
+            transform.Translate(Vector2.right * Time.deltaTime * _speed);
+        }
+        else
+        {
+            transform.Translate(Vector2.left * Time.deltaTime * _speed);
```

Comment:
Use ternary operator (research if you want) or use a method to abstract this translations into one method